x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/api.ts

```diff
@@ -135,18 +126,17 @@ export const fetchRuleById = async ({ id, signal }: FetchRuleProps): Promise<Rul
  *
  * @throws An error if response is not OK
  */
-export const enableRules = async ({ ids, enabled }: EnableRulesProps): Promise<Rule[]> => {
-  const response = await KibanaServices.get().http.fetch<Rule[]>(
+export const enableRules = async ({
+  ids,
+  enabled,
+}: EnableRulesProps): Promise<BulkRuleResponse> => {
+  return KibanaServices.get().http.fetch<BulkRuleResponse>(
```

Comment:
Is it possible to use an implicit return here?

Reply:
https://github.com/elastic/kibana/pull/59438/commits/9e367155b6988676e1a6252411a54186fc0dda4e